src/pages/SignIn.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react"
+import { firebaseApp } from "../lib/firebaseApp/index.js"
+
+console.log(firebaseApp)
+
+export const SignIn = () => {
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+
+  const submit = async () => {
+    alert(`
+      ${email}
+      ${password}
+    `)
+
+    try {
+      const res = await firebaseApp.auth().signInWithEmailAndPassword(email, password)
+      console.log("SUCCEEDED!")
+      console.log(res)
+
+      // uid使用できるか確認
+      const uid = res.user.uid
```

Comment:
欲しいのはid_tokenだったりするので
```suggestion
      firebase.auth().currentUser.getIdToken(false).then(idToken => console.log(idToken))
```
で確認する感じかも
https://firebase.google.com/docs/auth/admin/verify-id-tokens?hl=ja#retrieve_id_tokens_on_clients

Reply:
awaitだと
```ts
const idToken = await firebase.auth().currentUser.getIdToken(false)
```
って感じか